app/views/service_manual/service_manual_topic.html.erb

```diff
@@ -1,11 +1,5 @@
 <%= content_for :title, "#{@content_item.title} - Service Manual" %>
 
-<% content_for :before_content do %>
-  <div class="govuk-width-container">
-    <%= render partial: "service_manual/service_manual_breadcrumbs", breadcrumbs: @presenter.breadcrumbs %>
-  </div>
-<% end %>
```

Comment:
Why were the breadcrumbs deleted here? I think to replicate how it is on live you'd render the breadcrumbs here and add your document to https://github.com/alphagov/frontend/blob/8f4326c056248a0f6a1b90b69c61ce3ff9636a03/app/helpers/application_helper.rb#L27

Reply:
Have now changed the breadcrumbs to work like this.

Reply:
Thanks 👍